xdebug_var.c

```diff
@@ -2384,7 +2408,13 @@ void xdebug_var_export_fancy(zval **struc, xdebug_str *str, int level, int debug
 
 	if (debug_zval) {
 		if (Z_TYPE_P(*struc) >= IS_STRING && Z_TYPE_P(*struc) != IS_INDIRECT) {
-			xdebug_str_add(str, xdebug_sprintf("<i>(refcount=%d, is_ref=%d)</i>", (*struc)->value.counted->gc.refcount, Z_TYPE_P(*struc) == IS_REFERENCE), 1);
+			if (Z_TYPE_P(*struc) == IS_STRING && ZSTR_IS_INTERNED(Z_STR_P(*struc))) {
```

Comment:
As there are really only two variants of the format (one with `<i>` and one without), and both used more than once, it's probably a good idea to refactor this logic (the original lines 2386-2390, for this occurrnce) into its own function, as there is quite a lot of repetitiveness now.